brave-tracesamplers/pom.xml

```diff
@@ -7,9 +7,9 @@
     <version>3.3.1-SNAPSHOT</version>
   </parent>
   
-  <artifactId>brave-tracefilters</artifactId>
+  <artifactId>brave-tracesamplers</artifactId>
   <packaging>jar</packaging>
-  <name>brave-tracefilters</name>
+  <name>brave-tracesamplers</name>
```

Comment:
because if we add something inside that doesn't have a ZK dep, then we'd end up with a similar problem as if we put it in core (the problem of unnecessary deps).